data/remnant/remnant missions.txt

```diff
@@ -875,10 +879,63 @@ mission "Remnant: Technology Available"
+				goto continue
+			label thorough
+			`	The prefects nods in agreement. "We take our security very seriously."`
+			label continue
+			`	"Now, before we arrive at <destination>, pick a pattern for yourself. Something short - a second or two at most, and something memorable to you." The prefect turns away from you to install a small device on the side of one of your bridge consoles, and connects a few wires into the panel. "Now, insert your license here, and tap your pattern on the license." He looks away as you follow his directions. "Done? Good. When we approach <destination>, you will have a few options: For a normal approach when everything is nominal, tap your pattern on the license. If there is a threat following you in - an enemy ship or some other kind of danger you want the monitors to watch for, pinch or squeeze the license instead. And if you have a problem onboard, such as having been captured and forced to fly the ship in tap the license in a random pattern. Lastly, if you are having trouble landing due to injury or damage to the ship, just press briefly on the license. If you don't touch the license at all, the monitors will assume that you are either incapacitated or otherwise not in command of your ship. We will still attempt to let you land safely and provide assistance, but will also be standing ready to destroy you and block your approach if we think you are becoming a threat."`
+	on complete
+		conversation
+			`	As you enter the upper atmosphere of <destination>, the prefect gestures toward the small bracket he installed on your console. You insert your license, and tap the pattern you created. He checks something on his commlink and then nods. "Your license and clearance have been accepted, Captain <first>. Carry on with landing as normal."  You proceed with the approach and landing, with the prefect pointing out noteworthy features along the way. He also points out various approach paths, including prefered routes to take if you are being chased. "If you have a hostile threat on your tail, take this landing corridor; which will expose trailing ships to fire from these points. And if you have something sticking tight in your shadow, do a roll at this point or that point. That will expose them to our high-precision batteries."`
```

Comment:
Again, use the planet replacement.
"He checks something on his commlink" -> "He checks his commlink".
Typo fix for "preferred".
Removed a double space.
Broke this up into two paragraphs.
Removed the same comma as Wizrad.
Semicolon to comma (instead of colon as Wizrad suggested).
```suggestion
			`	As you enter the upper atmosphere of <planet>, the prefect gestures toward the small bracket he installed on your console. You insert your license and tap the pattern you created. He checks his commlink and then nods. "Your license and clearance have been accepted, Captain <first>. Carry on with landing as normal."`
			`	You proceed with the approach and landing, with the prefect pointing out noteworthy features along the way. He also points out various approach paths, including preferred routes to take if you are being chased. "If you have a hostile threat on your tail, take this landing corridor, which will expose trailing ships to fire from these points. And if you have something sticking tight in your shadow, do a roll at this point or that point. That will expose them to our high-precision batteries."`
```